LibreNMS/Alert/Transport/Api.php

```diff
@@ -52,10 +53,23 @@ private function contactAPI($obj, $api, $method)
     {
         $method = strtolower($method);
         list($host, $api) = explode("?", $api, 2);
+        $api = trim($api);
+
         foreach ($obj as $k => $v) {
-            $api = str_replace("%" . $k, $method == "get" ? urlencode($v) : $v, $api);
+            $api = str_replace("{{ $" . $k . " }}", $method == "get" ? urlencode($v) : $v, $api);
+        }
+
+        try {
+            $api = view(['template' => $api], $obj)->__toString();
```

Comment:
This is the Blades way of doing it, but this one does not urlencode in case method is GET

Reply:
solved now.